docs/support/known-issues.md

```
@@ -7,12 +7,43 @@ If you found an issue not mentioned here, please head over to our GitHub reposit
 If not feel free to open a new one.
 :::
 
+## refarch-gateway
+
+### "Origin not allowed" but configured
+
+By providing default CORS configuration for `/public/**` and `/clients/**` routes (to support configuration via environment variables), 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)